### PR TITLE
Functions for time durations

### LINKED ISSUE #136: Add Time Duration Utility
CSS animation durations can have ms or s suffixes to denote time.

For example:

```
animation-duration: 10s, 35s, 230ms;
```

Create a utility function in the [styles package utils file](https://github.com/chasefleming/elem-go/blob/main/styles/utils.go), alongside the other utilities, to handle these time options:

```
Seconds(10) // takes float64, returns string
Seconds(10.5) // takes float64, returns string
Milliseconds(230) // takes int
```

Ensure to include tests for these functions.

### PR DESCRIPTION
**Pull request by BreakTos**
closes #136

### CODE REVIEW

File: styles/utils.go [hunk 1]
```diff
@@ -118,3 +118,20 @@ func ViewportMin(value int) string {
 func ViewportMax(value int) string {
 	return strconv.Itoa(value) + "vmax"
 }
+
+// Seconds returns a string representation of the given float64 or int
+func Seconds(value interface{}) string {
```
Comment (chasefleming) [inline styles/utils.go:123]: I think you should just be able to take a `float64` as the value and return your float64 case. Then I believe `Seconds(10)` or Seconds(10.0)` would still return as `10.00s`.

File: styles/utils_test.go [hunk 2]
```diff
@@ -149,3 +150,17 @@ func TestViewportMax(t *testing.T) {
 	assert.Equal(t, "50vmax", ViewportMax(50))
 	assert.Equal(t, "25vmax", ViewportMax(25))
 }
+
+func TestSeconds(t *testing.T) {
+	assert.Equal(t, "100.00s", Seconds(100.00))
+	assert.Equal(t, "1.25s", Seconds(1.25))
+	assert.Equal(t, "0s", Rem(0))
```
Comment (chasefleming) [inline styles/utils_test.go:157]: Should be `Seconds` not `Rem` here.

File: styles/utils_test.go [hunk 3]
```diff
@@ -149,3 +150,17 @@ func TestViewportMax(t *testing.T) {
...
+	assert.Equal(t, "0.25s", Rem(0.25))
+}
+
+func TestMilliseconds(t *testing.T) {
+	assert.Equal(t, "25ms", Seconds(25))
+	assert.Equal(t, "1ms", Seconds(1))
+	assert.Equal(t, "0ms", Rem(0))
```
Comment (chasefleming) [inline styles/utils_test.go:164]: Same

**Review by chasefleming — CHANGES_REQUESTED** [review]
Looking good. Thanks for grabbing this. Just a couple minor changes.

File: styles/utils_test.go [hunk 4]
```diff
@@ -149,3 +150,17 @@ func TestViewportMax(t *testing.T) {
...
+func TestSeconds(t *testing.T) {
+	assert.Equal(t, "100.00s", Seconds(100.00))
+	assert.Equal(t, "1.25s", Seconds(1.25))
+	assert.Equal(t, "0s", Rem(0))
+	assert.Equal(t, "0.25s", Rem(0.25))
+}
+
+func TestMilliseconds(t *testing.T) {
+	assert.Equal(t, "25ms", Seconds(25))
+	assert.Equal(t, "1ms", Seconds(1))
```
Comment (chasefleming) [inline styles/utils_test.go:163]: Should be milliseconds

File: styles/utils.go (hunk 1 above)
Comment (BreakTos) [reply styles/utils.go:123]: I tried with float64 there, It wasn't accepting 10 as a parameter in that case and function overloading is apparently not possible in go

File: styles/utils_test.go (hunk 4 above)
Comment (BreakTos) [reply styles/utils_test.go:163]: oopsies late night copy pasting, just a second

File: styles/utils.go (hunk 1 above)
Comment (chasefleming) [reply styles/utils.go:123]: I can confirm it works for `Rem`. Just tried adding these test cases: https://github.com/chasefleming/elem-go/pull/138/files#diff-97855a388ba6def6d1f94ab2c728bead74eb8af8ac30d1a2d6e09c5829336ac8R87

Comment (BreakTos) [reply styles/utils.go:123]: I see, What i tried was first creating a variable and then passing which was stemming the confusion
image

Directly passing the integer comes across as float. Thanks

Comment (chasefleming) [reply styles/utils.go]: Yeah, when you define a variable with the shorthand it is inferring the type as int. You'd need to declare it with a float64 type to pass it in from a variable.

Comment (BreakTos) [reply styles/utils.go]: ha.. fixed in the latest commit. Is it fine to be merged ?

**Review by chasefleming — APPROVED** [review]
Looks good @BreakTos . Thanks for taking this one.

**Pull request merged**